lntest/itest/assertions.go

```diff
@@ -443,10 +455,14 @@ type lnrpcForceCloseChannel = lnrpc.PendingChannelsResponse_ForceClosedChannel
 
 // waitForNumChannelPendingForceClose waits for the node to report a certain
 // number of channels in state pending force close.
-func waitForNumChannelPendingForceClose(ctx context.Context,
-	node *lntest.HarnessNode, expectedNum int,
+func waitForNumChannelPendingForceClose(node *lntest.HarnessNode,
+	expectedNum int,
 	perChanCheck func(channel *lnrpcForceCloseChannel) error) error {
 
+	ctxb := context.Background()
+	ctx, cancel := context.WithTimeout(ctxb, channelCloseTimeout)
```

Comment:
Same here and below re default timeout vs. channel close timeout.

Reply:
Done!